things/postgres/init.go

```diff
@@ -93,6 +93,32 @@ func migrateDB(db *sqlx.DB) error {
 					 metadata TYPE JSONB using metadata::text::jsonb`,
 				},
 			},
+			{
+				Id: "things_4",
+				Up: []string{
+					`ALTER TABLE IF EXISTS things ADD CONSTRAINT things_id_key UNIQUE (id)`,
+					`CREATE extension LTREE`,
+					`CREATE TABLE IF NOT EXISTS thing_groups ( 
+						id          UUID UNIQUE NOT NULL,
+						parent_id   UUID, 
+						owner_id    UUID,
+						name        VARCHAR(254) NOT NULL,
+						description VARCHAR(1024),
+						metadata    JSONB,
+						path        LTREE, 
+						PRIMARY KEY (name, path),
```

Comment:
Does it make sense to have an ID that's not a primary key?

Reply:
https://github.com/mainflux/mainflux/pull/1259#discussion_r525561057 Im talking about json response